tools/sof-kernel-log-check.sh

```diff
@@ -40,6 +25,30 @@ ignore_str="$ignore_str"'|PM: Device i2c-DELL0955:00 failed to resume async: err
 # https://github.com/thesofproject/sof-test/issues/348
 ignore_str="$ignore_str"'|da7219 i2c-DLGS7219:00: SRM failed to lock'
 
+# Dell CML-U laptop with SoundWire, issues reported by sof-test
+# https://github.com/thesofproject/sof-test/issues/343
+ignore_str="$ignore_str"'|tpm tpm0: tpm_try_transmit: send(): error -5'
+ignore_str="$ignore_str"'|platform regulatory.0: Direct firmware load for regulatory.db failed with error -2'
+ignore_str="$ignore_str"'|cfg80211: failed to load regulatory.db'
+ignore_str="$ignore_str"'|EXT4-fs (nvme0n1p6): re-mounted. Opts: errors=remount-ro'
```

Comment:
Are you sure this works? Parentheses `( )` are special characters in Extended regular expressions:

```shell
printf '%s' 'h(ell)o' | grep   'h(ell)o'
h(ell)o # match OK
printf '%s' 'h(ell)o' | grep -E 'h(ell)o'
# nothing, no match!
```
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_04

If lines with parentheses fail to filter as I'm afraid they do, then you can simply replace them with dots `.`

Reply:
it doesn't, we need to use escape characters. fixed now.